### PR TITLE
Add Legacy code, Fix formatting of output/input examples

### PR DESCRIPTION
**Pull request by lettucegoblin**
@iron768 is responsible for:
100% of src/main/java/org/legacy/BigIntRectangle.java
100% of integration of BigInt in src/main/java/org/legacy/FibonacciFractalGenerator.java

### CODE REVIEW

**Review by ndg8743 — COMMENTED** [review]
Yup protype code is looking good
Great success

File: legacy/BigIntRectangle.java
Comment (iron768) [inline legacy/BigIntRectangle.java:1]: checkstyle review: line 1 uses * import, line 7 and 8 variables are named x and y

File: legacy/FibonacciFractalGenerator.java
Comment (iron768) [inline legacy/FibonacciFractalGenerator.java:1]: checkstyle review: line 2 has a *, line 82 switch without default, line 110 has a if statement without braces, lines 193 and 194 have the variables x and y and the checkstyle doesnt like that

**Review by iron768 — COMMENTED** [review]
Code looks correct, any issues with checkstyle were marked on the appropriate Java files.

File: legacy/BigIntRectangle.java
Comment (lettucegoblin) [reply legacy/BigIntRectangle.java:1]: Thank you!

**Review by ndg8743 — COMMENTED** [review]
Epic

**Review by ndg8743 — COMMENTED** [review]
Even greater success, code changed to align with checkstyle rules

**Review by iron768 — APPROVED** [review]
Code is corrected!

**Pull request merged**